### ISSUE TITLE
Versions <= 3.52.0.99378 Relaunch DeviceRootedActivity

### ISSUE THREAD

**Issue by corndoghh** [corndoghh/Starling-Bypass#2]
Starling Versions above and including 3.52.0.99378 appear to relaunch the DeviceRootedActivity every time another activity attempts to gain TopActivity.

Rendering this module redundant for versions above 3.51.0.99046

For now manually downgrade to 3.51.0.99046 to bypass the DeviceRootedActivity

**Comment by SquadCraftFTW** [comment]
3.51.0.99046 no longer supported :/

**Comment by DupeDawn** [comment]
> 3.51.0.99046 no longer supported :/

Yea this tool is great but because starling is so over the top I've switched to Monzo now. Simply because the developer/owner has said they see no point In locking out rooted devices, but if the user has money stolen due to being rooted they won't help.

**Comment by SquadCraftFTW** [comment]
I use them because of their virtual cards, not many other banks with virtual cards let you make new ones as frequently as starling

**Comment by corndoghh** [comment]
> > 3.51.0.99046 no longer supported :/
> 
> Yea this tool is great but because starling is so over the top I've switched to Monzo now. Simply because the developer/owner has said they see no point In locking out rooted devices, but if the user has money stolen due to being rooted they won't help.

Yep starling is very over the top, i`m still using them now as it will be a hassle to switch

i'm using kernelSU as a rooting method now which starling cant detect as of yet.
 
However I don`t think this alone would be enough if you use custom ROMs
